playbooks/tests/tasks/lbaas.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: tests lbaas
+  hosts: controller[0]
+  vars:
+    image_name: cirros
+    flavor_name: m1.tiny
+    pub_net: external
+    pri_net: internal
+    sub_net: internal_v4
+    stack_name: test_lbaas
+  tasks:
+  - copy: src={{ lookup('env', 'URSULA_ENV') }}/../..//playbooks/tests/tasks/lb_stack.yml dest=/tmp/lb_stack.yml owner=ubuntu group=ubuntu
+  - name: make sure admin has no role - heat_stack_user
+    shell: . /root/stackrc; openstack role add --user admin --project admin heat_stack_user; openstack role remove --user admin --project admin heat_stack_user;
```

Comment:
This will result in a failure if the user doesn't have that role. The command will exit non-zero.

The user shouldn't have that role, and if it does have that role, we've made some mistake in the Ursula code to get to this point, so we shouldn't manipulate things like this.